vision/google/cloud/vision/color.py

```diff
@@ -149,19 +183,34 @@ def __init__(self, color, score, pixel_fraction):
         self._pixel_fraction = pixel_fraction
 
     @classmethod
-    def from_api_repr(cls, response):
-        """Factory: construct ``ColorInformation`` for a color found.
+    def from_api_repr(cls, color_information):
+        """Factory: construct ``ColorInformation`` for a color.
 
-        :type response: dict
-        :param response: Color data with extra meta information.
+        :type color_information: dict
+        :param color_information: Color data with extra meta information.
 
         :rtype: :class:`~google.cloud.vision.color.ColorInformation`
         :returns: Instance of ``ColorInformation``.
         """
-        color = Color.from_api_repr(response.get('color'))
-        score = response.get('score')
-        pixel_fraction = response.get('pixelFraction')
+        color = Color.from_api_repr(color_information.get('color'))
```

Comment:
What happens if `Color.from_api_repr` gets `None`? Don't you want to fall back to a dict?